hugo/content/faqs/_index.md

```diff
@@ -2,87 +2,88 @@
 title: Frequently Asked Questions
 ---
 
-[faq]
-title = "FAQs"
+### FAQs
 
-## About the Genome Portal
+#### About the Genome Portal
 
-[[faq.section]]
-question = "What is the Swedish Reference Genome Portal?"
-answer = "TBD"
+{{< faq_block title="How does the Swedish Reference Genome Portal differ from other genome portal initiatives? (e.g., Ensembl, UCSC)" >}}
+The Swedish Reference Genome Portal (SRGP) does not replace existing genome portal initiatives, as it has specific aims and scope:
 
-[[faq.section]]
-question = "How does the Genome Portal differ from existing genome browsers such as Ensembl, USCS, JGI, IGV, among others?"
-answer = "TBD"
+- The SRGP targets the Swedish research community and is restricted to non-human eukaryotic species. Therefore, the species displayed here are typically limited to those submitted by researchers affiliated with Swedish institutions. In contrast, global genome portals such as Ensembl and UCSC have much broader taxonomic scope, aiming to include species from around the world, including humans and bacteria.
 
-[[faq.section]]
-question = "What are the benefits of using the Genome Portal?"
-answer = "TBD"
+- The SRGP's primary goal is to facilitate access, visualisation, and interpretation of genomic data, with a focus on offering a powerful genome browser. Global genome portals like Ensembl and UCSC offer additional features including those related to comparative genomics, besides genome browser capabilities.
 
-[[faq.section]]
-question = "What are the minimal requirements to display genomic data on the Genome Portal?"
-answer = "TBD"
+- The SRGP is unique in that it allows researchers to decide what, when, and how their genomic data is displayed on the Genome Portal. While global genome portals can also address researchers' inquiries, users can generally expect much shorter processing times with the SRGP, as it is maintained by a local team.
```

Comment:
Since do have a minimal requirement on the contribute page, let's make sure that we don't say different things in different pages 😄 

```suggestion
- The minimal requirement is that the species has a genome assembly in FASTA file format and an annotation of the protein-coding genes, preferably in GFF format. Other than that, the SRGP is unique in that it allows researchers to decide what, when, and how their genomic data is displayed on the Genome Portal. While global genome portals can also address researchers' inquiries, users can generally expect much shorter processing times with the SRGP, as it is maintained by a local team.